icu4c/source/common/unicode/locid.h

```diff
@@ -296,6 +304,16 @@ class U_COMMON_API Locale : public UObject {
      */
     Locale& operator=(const Locale& other);
 
+    /**
+     * Replaces the entire contents of *this with the specified value.
```

Comment:
Move assignment operator; might leave src in bogus state.

This locale will have the same contents that the source locale had. The behavior is undefined if *this and src are the same object.

Reply:
Done.